manifests/c/com/jmmgc/ocwin/18.1/com.jmmgc.ocwin.yaml

```
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.10.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.10.0.schema.json
+
+PackageIdentifier: com.jmmgc.ocwin

```

```suggestion
PackageIdentifier: jmmgc.ocwin
```
Please update file name and path to match this change.